src/jws.rs

```diff
@@ -250,6 +250,7 @@ pub fn verify_bytes_warnable(
     key: &JWK,
     signature: &[u8],
 ) -> Result<VerificationWarnings, Error> {
+    #[allow(unused_mut)]
     let mut warnings = VerificationWarnings::default();
```

Comment:
is the `mut` necessary?

Reply:
It is if the `k256` feature is enabled, currently for ES256K and ES256K-R:
https://github.com/spruceid/ssi/blob/1d608dde99770ee839b5e3702599402cc9601c0d/src/jws.rs#L364-L367
https://github.com/spruceid/ssi/blob/1d608dde99770ee839b5e3702599402cc9601c0d/src/jws.rs#L390-L391